LibVLCSharp/Shared/MediaPlayer.cs

```diff
@@ -8,7 +8,7 @@ namespace LibVLCSharp.Shared
 {
     public class MediaPlayer : Internal
     {
-        struct Native
+        readonly struct Native
```

Comment:
Why are all "Native" `readonly struct`? Wouldn't it make sense to have them `internal static class` instead?

Reply:
Putting the dllimport statements in a private struct is a cppsharp convention which I find OK. What advantage would bring using a static class for those?

Reply:
The only two reasons are :

- `static class` conveys the meaning that it does not name a variable type, but only contains helpers
- I've always done it that way.

Otherwise, if you say that it's a cppsharp convention, I'm fine with this. I don't know the technical reasons over a choice rather than another.

Reply:
I also agree with @jeremyVignelles. The small inconvenient with `struct` is that you can instantiate it (you can do `new Native()`, which does not really make sense here).

Reply:
> you can do new Native()

Uh... sure :D but 3rd party users cannot do that, only people editing libvlcsharp.

It's not a very impactful change IMO, but I will think about it and make another PR if we go the `static class` route.